server/storage/interface.go

```diff
@@ -39,4 +39,15 @@ type MetaStore interface {
 	// Delete removes all metadata for a given GUN.  It does not return an
 	// error if no metadata exists for the given GUN.
 	Delete(gun string) error
+
+	// GetChanges returns an ordered slice of changes. It starts from
+	// the change matching changeID, but excludes this change from the results
+	// on the assumption that if a user provides an ID, they've seen that change.
+	// If changeID is 0, it starts from the
```

Comment:
I think we an probably safely say that `-1` will never be a valid ID so we just keep that as the official "latest" indicator. The literal characters in the query string don't change between `-1` the integer, and `-1` the string, it's up to the server to interpret it.